logstash-core/src/main/java/org/logstash/config/ir/graph/DiffResult.java

```diff
@@ -0,0 +1,75 @@
+package org.logstash.config.ir.graph;
```

Comment:
Thanks for extracting this into his own class, I know this is mostly for debugging but can we have 2-3 unit test to make sure we don't regress it? I am pretty sure we improve this classe for other stuff.